qmk_commands.py

```diff
@@ -28,6 +28,8 @@
 CHIBIOS_GIT_URL = os.environ.get('CHIBIOS_GIT_URL', 'https://github.com/qmk/ChibiOS')
 CHIBIOS_CONTRIB_GIT_BRANCH = os.environ.get('CHIBIOS_CONTRIB_GIT_BRANCH', 'qmk')
 CHIBIOS_CONTRIB_GIT_URL = os.environ.get('CHIBIOS_CONTRIB_GIT_URL', 'https://github.com/qmk/ChibiOS-Contrib')
+PRINTF_GIT_BRANCH = os.environ.get('PRINTF_GIT_BRANCH', 'qmk')
```

Comment:
Shouldn't this be

```suggestion
PRINTF_GIT_BRANCH = os.environ.get('PRINTF_GIT_BRANCH', 'master')
```

?

Reply:
fuuu yeah